lib/improver/nowcasting/optical_flow.py

```diff
@@ -248,9 +255,6 @@ def process(self, cube, timestep, fill_value=0.0):
                 The 2D cube containing data to be advected
             timestep (datetime.timedelta):
                 Advection time step
-            fill_value (float):
-                Default output value for spatial points where data cannot be
-                extrapolated (source is out of bounds)
 
         Returns:
             advected_cube (iris.cube.Cube):
```

Comment:
I think the doc for the returned "advected_cube" should be updated to specify that points where source was out-of-bounds in the extrapolation will be returned in a masked state. Currently, this info is buried in the doc-string of a hidden method.